README.md

```diff
@@ -35,23 +35,137 @@ nil: {"nil"=>41, "redhat_7.2"=>5}
 - For each run list combination download all the cookbooks in that run list from the server into unique directories on the local host
 - Run cookstyle / foodcritic against those directories alerting on just the deprecation rules. Create a report of code changes that will need to be made per run list combination
 
+## Migration Pattern
 
-## Comparison - Chef Infra | Effortless
+### Current Production Cookbooks
+1. Gather list of production cookbooks.
+1. Pull down first cookbook for local testing.
+1. Create repo in SCM if the cookbook doesn't already have it's own repo (1-per-cookbook), commit cookbook to SCM before changes are made (if already in SCM, verify diff).
+1. Create new Chef-15 branch for the repo locally, switch to that branch for all further work.
+1. Run Cookstyle, Foodcritic + other tools on cookbook.
+1. Assess the changes needed.
+1. Confirm changes required, make manual changes if needed.
+1. Some changes will have the possiblity to be auto-remediated.
+1. Continue process until all needed changes to have cookbook working with Chef Infra Client 15 are remediated and the cookbook tests successfully in Test Kitchen.
+1. Add Habitat Plan to cookbook.
+1. Commit changes to repo.
+1. Commit will trigger build webhook to fire, the Habitat Package will be built and stored in Hab Depot via job server.
+1. The job server will also perform reverse dependency lookup and pull down all cookbooks that the primary cookbook depends upon, all the way up the chain.
+1. The job server will determine if there are any attributes referenced that are dependent upon envrionments (either local or Chef Infra Envrionment attributes), as well as any references to data bags.
+1. Each cookbook will be packaged into it's own Habitat pkg.
+1. The end result will be a Habitat Packaged cookbook stored in Hab Depot, along with a manifest of all newly created dependency cookbook packages it depends upon (stored in package and as output) as well as a manifest of all Environment-level attributes and Data Bag references.
```

Comment:
This dependency list for cookbooks and attributes would be reflected in the packages listed in the the plan right? Do we need another manifest?

Reply:
Would like the plan to be the source of truth of what the dependencies are, is there something we would need to change in the plan to avoid using other manifests?